src/FRC.NetworkTables/TcpSockets/TcpConnector.cs

```diff
@@ -26,7 +26,10 @@ private static int ResolveHostName(string hostName, out IPAddress[] addr)
                     // Sockets don't all support IPV6
                     if (ipAddress.AddressFamily == AddressFamily.InterNetwork)
                     {
-                        addresses.Add(ipAddress);
+                        if (!addresses.Contains(ipAddress))
+                        {
+                            addresses.Add(ipAddress);
+                        }
```

Comment:
Interesting. Can you split this change into a separate PR then? I'd like to merge this separately from the rest of the changes.